contents/docs/product-analytics/troubleshooting.mdx

```diff
@@ -117,6 +117,10 @@ To confirm if this is the issue, use the following steps:
 
 Once you have confirmed there is an issue with event ingestion, you can [report your issue](#report-your-issue) to get further support.
 
+## How do I rename my events?
+
+It's not possible to rename events in PostHog. Instead, you should use [actions](add link) to xyz. This [tutorial](/tutorials/how-to-rename-events) covers this in detail.
```

Comment:
needs editing 😅

Reply:
Oh man nice catch. I was so focused on the other file that I almost forgot about this link. Fixed in https://github.com/PostHog/posthog.com/pull/9703/commits/7269c8ca6ab11e73749f74aaacfa0d855da694ac.